packages/ui-kit/src/helpers/graphql/public.graphql

```diff
@@ -2727,6 +2731,10 @@ input TimeSeriesInput {
   """
   metric: MetricInput
   """
+  The Metric to query. You can query a pre-configured Metric by ID or name, or you can query an ad hoc Metric that you define inline.
+  """
+  metric: MetricInput
```

Comment:
I'm confused. Doesn't this look duplicated? AFAICT, this file should not be changed at all?